Formula/gtk+3.rb

```diff
@@ -119,5 +124,21 @@ def post_install
     ]
     system ENV.cc, "test.c", "-o", "test", *flags
     system "./test"
+    # include a version check for the pkg-config files
+    assert_match version.to_s, shell_output("cat #{lib}/pkgconfig/gtk+-3.0.pc").strip
```

Comment:
I'm not sure that we need to add test cases for upstream bugs

Reply:
I would make an exception for this one, as this seems to be a recurring one, and brings a lot of confusion for some linux builds that will complain about version mismatches.